libvast/builtins/formats/zeek_tsv.cpp

```diff
@@ -670,6 +666,12 @@ auto parser_impl(generator<std::optional<std::string_view>> lines,
         co_return;
       }
     }
+    if (b->rows() > defaults::import::table_slice_size
+        or last_finish + std::chrono::seconds{1}
+             < std::chrono::steady_clock ::now()) {
+      last_finish = std::chrono::steady_clock::now();
```

Comment:
I don't think the compiler will be able to merge these calls. Can you change it so it only makes the call once and assigns the result?

Reply:
Right, fair point.

Reply:
Done in 662725466a